test/src/ui/js/test_Game.js

```diff
@@ -1702,16 +1864,17 @@ test("test_Game.dieBorderTogglePlayerHandler", function(assert) {
     // and unselected on click
     var dieobj = $('#playerIdx_0_dieIdx_0');
     var html = $('<div>').append(dieobj.clone()).remove().html();
-    assert.ok(html.match('die_container die_alive unselected_player'),
+    assert.ok(html.match('die_container die_alive hide_focus unselected_player'),
```

Comment:
Are there any side effects to always specifying hide_focus when we didn't before?  I'll keep an eye out for that in testing.